src/components/pages/Settings/Expansions/ContentCustomization/index.tsx

```diff
@@ -35,19 +36,93 @@ const mapStateToProps = (state: RootState, { expansionId }: OwnProps) => {
       state,
       { expansionId }
     ),
+    allNemesesSelected: selectors.Settings.Expansions.Nemeses.getAllNemesesOfExpansionSelected(
+      state,
+      { expansionId }
+    ),
+    allMagesSelected: selectors.Settings.Expansions.Mages.getAllMagesOfExpansionSelected(
+      state,
+      { expansionId }
+    ),
+    allSupplyCardsSelected: selectors.Settings.Expansions.Cards.getAllCardsOfExpansionSelected(
+      state,
+      { expansionId }
+    ),
+    allTreasuresSelected: selectors.Settings.Expansions.Treasures.getAllTreasuresOfExpansionSelected(
+      state,
+      { expansionId }
+    ),
+    allBasicNemesisCardsSelected: selectors.Settings.Expansions.BasicNemesisCards.getAllBasicNemesisCardsOfExpansionSelected(
+      state,
+      { expansionId }
+    ),
+    allUpgradedBasicNemesisCardsSelected: selectors.Settings.Expansions.UpgradedBasicNemesisCards.getAllUpgradedBasicNemesisCardsOfExpansionSelected(
+      state,
+      { expansionId }
+    ),
   }
 }
 
 const mapDispatchToProps = {
   // FIXME use available expansions instead
   selectLanguage: actions.Settings.Expansions.Languages.select,
+  handleSelectAllNemeses: actions.Settings.Expansions.Nemeses.main.toggleAll,
+  handleSelectAllMages: actions.Settings.Expansions.Mages.main.toggleAll,
+  handleSelectAllSupplyCards: actions.Settings.Expansions.Cards.main.toggleAll,
+  handleSelectAllTreasures:
+    actions.Settings.Expansions.Treasures.main.toggleAll,
+  handleSelectAllBasicNemesisCards:
+    actions.Settings.Expansions.BasicNemesisCards.main.toggleAll,
+  handleSelectAllUpgradedBasicNemesisCards:
+    actions.Settings.Expansions.UpgradedBasicNemesisCards.main.toggleAll,
 }
 
 type Props = ReturnType<typeof mapStateToProps> &
   typeof mapDispatchToProps &
   OwnProps
 
-const ContentCustomization = ({ expansionId, lang, selectLanguage }: Props) => {
+const ContentCustomization = ({
+  expansionId,
+  lang,
+  allNemesesSelected,
+  allMagesSelected,
+  allSupplyCardsSelected,
+  allTreasuresSelected,
+  allBasicNemesisCardsSelected,
+  allUpgradedBasicNemesisCardsSelected,
+  selectLanguage,
+  handleSelectAllNemeses,
+  handleSelectAllMages,
+  handleSelectAllSupplyCards,
+  handleSelectAllTreasures,
+  handleSelectAllBasicNemesisCards,
+  handleSelectAllUpgradedBasicNemesisCards,
+}: Props) => {
+  const handleCheckboxChange = () => {
+    if (
+      allNemesesSelected &&
+      allMagesSelected &&
+      allSupplyCardsSelected &&
+      allTreasuresSelected &&
+      allBasicNemesisCardsSelected &&
+      allUpgradedBasicNemesisCardsSelected
```

Comment:
we could combinethese into a single `all...Selected` selector :)

Reply:
done :)